components/smalltalk/squeak4/squeak4.p5m

```diff
@@ -0,0 +1,49 @@
+set name=pkg.fmri value=pkg:/runtime/squeak4@$(IPS_COMPONENT_VERSION)
+set name=pkg.description value="Squeak V4"
+set name=pkg.summary value="The Squeak V4 Smalltalk System"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+
```

Comment:
It looks like you have completely hand-crafted this .p5m file.
Usually you should run "gmake sample-manifest" that generates a stub file in the manifests folder.
This file can be used as an input to pkgmogrify (which does some automatic transformations),
optionally add some more transformation rules to apply and finally piped into pkgfmt.
This will generate an almost complete p5m file which typically only need a few manual adaptions.

Reply:
The p5m is not entirely hand-crafted.   I started from a "pkgsend generate" output of the prototype directory, which I then started to edit in the p5m file until I had something that "gmake publish" accepts.

Reply:
It's better to rely on "gmake sample-manifest" and add/change whatever is needed then.

Reply:
Running gmake sample-manifest generates a simpler file which you can take as a starting point.
Copying over some additional lines from your original p5m file would be easy.